api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -352,6 +360,28 @@ def get_stats(venue_id: int) -> utils.BackofficeResponse:
     )
 
 
+@venue_blueprint.route("/<int:venue_id>/delete/<int:provider_id>", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.MANAGE_PRO_ENTITY)
+def delete_venue_provider(venue_id: int, provider_id: int) -> utils.BackofficeResponse:
+    provider = providers_models.Provider.query.filter_by(id=provider_id).one_or_none()
+
+    if provider and provider.localClass == "AllocineStocks":
+        flash("Impossible d'effacer le lien entre le lieu et Allocine.", "warning")
+        return redirect(url_for("backoffice_v3_web.venue.get", venue_id=venue_id), code=303)
+
+    result = providers_models.VenueProvider.query.filter(
+        providers_models.VenueProvider.providerId == provider_id,
+        providers_models.VenueProvider.venueId == venue_id,
+    ).delete()
+
+    db.session.commit()
```

Comment:
Si `not result`, le `commit()` n'a probablement pas de sens, on pourrait le déplacer dans le `if`.

Reply:
Pas d'accord ou oubli ?

Reply:
je l'ai pas fait ? si c'est le ca oubli dsl

Reply:
corrigé